app/assets/javascripts/preview.js

```diff
@@ -0,0 +1,25 @@
+$(document).on("turbolinks:load", function () {
+  $("#card-preview").click(function(){
+    previewCard();
+    highlight();
+  });
+});
+
+function previewCard() {
+  marked.setOptions({
+    renderer: new marked.Renderer(),
+    gfm: true,
+    tables: true,
+    breaks: true,
+    pedantic: false,
+    sanitize: false,
+    smartLists: true,
+    smartypants: false
+  });
+
+  var text = $("#card-description").val();
```

Comment:
Yup.
Or even live the `if` but extract the `html` call could be beneficial for the reading? Or is just a matter of taste?

```js
preview = text ? makred(text) : "Nothing to preview";
$(".preview").html(preview);
```